### PR TITLE
Quarkus 0.20.0 announcement post

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: _posts/2019-07-31-quarkus-0-20.0-released.adoc [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+ * `quarkus-jackson` for standalone usage of Jackson
+ * `quarkus-resteasy-jackson` to use Jackson for your REST services
+
+==== OAuth2 extension
+
+The new Security OAuth2 extension allows to implement an authentication mechanism based on OAuth2 opaque tokens. 
+It complements the existing JWT extension by allowing to authenticate a user based on a call to an introspection endpoint instead of decoding the token. 
+An opaque token is a token with no information encoded in it, so it needs an external call to gather the user information, this external call is made against what is called an introspection endpoint.
+
+Behind the scene, it uses Elytron and the same annotations as all the other security extensions, so it's as easy to use as the other authentication mechanism that Quarkus supports, and a user can easily switch between them if needed.
```
Comment (geoand) [inline _posts/2019-07-31-quarkus-0-20.0-released.adoc]: Should it be `Behind the scenes`?

Comment (geoand) [reply _posts/2019-07-31-quarkus-0-20.0-released.adoc]: Also `it uses Elytron` might be better as `Elytron is used`

Comment (gsmet) [reply _posts/2019-07-31-quarkus-0-20.0-released.adoc]: Fixed the first one. As for the second one, it doesn't work with the rest of the sentence so I kept it as is.

Comment (geoand) [reply _posts/2019-07-31-quarkus-0-20.0-released.adoc]: :+1:

File: _posts/2019-07-31-quarkus-0-20.0-released.adoc [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+date: 2019-07-31
+tags: quarkus release
+author: gsmet
+---
+
+We just released Quarkus 0.20.0.
+
+[WARNING]
+====
+Quarkus 0.20.0 requires GraalVM 19.1.1 so please upgrade your GraalVM installation.
```
Comment (machi1990) [inline _posts/2019-07-31-quarkus-0-20.0-released.adoc]: I would expect this to only be valable if you are building a native executable.

Comment (geoand) [reply _posts/2019-07-31-quarkus-0-20.0-released.adoc]: I think this is same wording that was used in previous announcements, correct @gsmet ?

**Pull request merged**